guides/common/modules/con_kickstart-workflow.adoc

```diff
@@ -14,4 +14,4 @@ When you run a {ProjectName} Kickstart script, the script performs the following
 . It runs user defined snippets.
 
 .Additional resources
-For more information about Kickstart, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in _Performing an advanced RHEL 8 installation_.
+For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_.
```

Comment:
@Lennonka I just created a [bug](https://issues.redhat.com/browse/SAT-30066) to update these links for RHEL 9. Just wanted to check -- was there any reason for using RHEL 8 links?

Reply:
If it's about hosts then there should be links to all supported RHEL versions, not only RHEL 8, which is one of the supported client OSs. I hope that helps.